operations/helm/charts/mimir-distributed/templates/_helpers.tpl

```diff
@@ -74,6 +74,18 @@ Create the name of the service account
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create the name of the ruler service account
+*/}}
+{{- define "mimir.ruler.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
```

Comment:
shouldn't this use the `.Values.ruler.serviceAccount.create` instead of `.Values.serviceAccount.create`?

Reply:
yeah you're right. Let me fix that.